### PR TITLE
chore: configure the automatic browser roll PRs

### PR DESCRIPTION
**Pull request by OrKoN**
For all configuration options see https://github.com/peter-evans/create-pull-request#action-inputs

### CODE REVIEW

File: .github/workflows/update-browser-version.yml [hunk 1]
```diff
@@ -32,3 +32,8 @@ jobs:
           commit-message: Update the browser version
           committer: Browser Automation Bot <browser-automation-bot@google.com>
           author: Browser Automation Bot <browser-automation-bot@google.com>
+          commit-message: "chore: update the pinned browser version"
```
Comment (OrKoN) [inline .github/workflows/update-browser-version.yml:35]: or should we use a non-conventional format?

Comment (thiagowfx) [reply .github/workflows/update-browser-version.yml]: We don't have any convention anyways. Anything is fine. Please note you have two `commit-message` entries, currently.

File: .github/workflows/update-browser-version.yml [hunk 2]
```diff
@@ -29,6 +29,10 @@ jobs:
         uses: peter-evans/create-pull-request@v5
         with:
           token: ${{ secrets.BROWSER_AUTOMATION_BOT_TOKEN }}
-          commit-message: Update the browser version
+          branch: create-pull-request/browser-update
```
Comment (thiagowfx) [inline .github/workflows/update-browser-version.yml:32]: I'd put the name of the bot here: `<bot>/<workflow name>`, so: `browser-automation-bot/update-browser-version`.

Comment (thiagowfx) [reply .github/workflows/update-browser-version.yml:32]: This matches our convention to use the author as the first branch name part

File: .github/workflows/update-browser-version.yml [hunk 3]
```diff
@@ -29,6 +29,10 @@ jobs:
...
+          delete-branch: true
           committer: Browser Automation Bot <browser-automation-bot@google.com>
           author: Browser Automation Bot <browser-automation-bot@google.com>
+          commit-message: 'chore: update the pinned browser version'
+          title: 'chore: update the pinned browser version'
+          body: 'Automatically generated by https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/update-browser-version.yml'
```
Comment (thiagowfx) [inline .github/workflows/update-browser-version.yml:38]: https://github.com/GoogleChromeLabs/chromium-bidi/pulls?q=is%3Apr+dependabot+is%3Aclosed+label%3Adependencies

Let's also add

```
labels: dependencies
```

to mirror dependabot

File: .github/workflows/update-browser-version.yml (hunk 1 above)
Comment (OrKoN) [reply .github/workflows/update-browser-version.yml]: I think I will use `build(chrome): ` to match dependabot too.

File: .github/workflows/update-browser-version.yml (hunk 3 above)
Comment (OrKoN) [reply .github/workflows/update-browser-version.yml:38]: Added

File: .github/workflows/update-browser-version.yml (hunk 1 above)
Comment (thiagowfx) [reply .github/workflows/update-browser-version.yml]: SG

**Review by thiagowfx — APPROVED** [review]

**Pull request merged**